services/metadata/conf/main/logback.xml

```diff
@@ -11,7 +11,7 @@
   </appender>
 
 
-  <Logger name="donkey" level="debug" />
+  <Logger name="terrain" level="debug" />
```

Comment:
Why is it we have configuration for the terrain logger in all of our services anyway? Presumably this should never be relevant except for terrain itself